src/qae/initial-properties.js

```diff
@@ -117,34 +117,98 @@ const properties = {
  */
 
 /**
- * General styling for all columns.
- * Split up into header and content (body) styling.
- * If any property is not set, default values specific for each property is used.
+ * Styling defintions
  * @name Component
+ * @type {ChartStyling | GeneralStyling}
+ */
+
+/**
+ * Mandatory key for general styling
+ * @name GeneralStylingKey
+ * @type {"general"}
+ */
+
+/**
+ * Font styling values
+ * @name FontStyleValues
+ * @type {"bold" | "italic" | "underline"}
+ */
+
+/**
+ * General chart styling
+ * @name GeneralStyling
+ * @type object
+ * @property {GeneralStylingKey} key
+ * @property {TitleStyling} title
+ */
+
+/**
+ * Title styling options
+ * @name TitleOptions
+ * @type object
+ * @property {string} [fontSize] - Font size in pixel value
+ * @property {string} [fontFamily] - Font family
+ * @property {FontStyleValues[]} [fontStyle] - Font style
+ * @property {PaletteColor} [color] - Font color palette
+ */
+
+/**
+ * Title styling
+ * @name TitleStyling
+ * @type object
+ * @property {TitleOptions} [main] - Styling for chart title
+ * @property {TitleOptions} [subTitle] - Styling for chart sub title
+ * @property {TitleOptions} [footer] - Styling for chart footer
+ */
+
+/**
+ * Mandatory key for chart styling
+ * @name ChartStylingKey
+ * @type {"theme"}
+ */
+
+/**
+ * Custom styling of cells
```

Comment:
Maybe change to just `Chart styling`?

Reply:
yeah I think so, the grid stuff is not really applied to independent cells